contracts/src/CrossChainApplications/NativeTokenBridge/NativeTokenSource.sol

```diff
@@ -18,9 +19,16 @@ import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol
 contract NativeTokenSource is
     ITeleporterReceiver,
     INativeTokenSource,
+    ITokenSource,
     ReentrancyGuard
 {
+    // The address where the burned transaction fees are credited.
+    address public constant BURNED_TX_FEES_ADDRESS =
+        0x0100000000000000000000000000000000000000;
     uint256 public constant MINT_NATIVE_TOKENS_REQUIRED_GAS = 100_000;
+    // Used to keep track of tokens burned through transactions on the destination chain. They can
+    // be reported to this contract to burn an equivalent number of tokens on this chain.
+    uint256 public destinationChainBurnedBalance = 0;
```

Comment:
IMO "Burned Amount" would be more accurate than "Burned Balance" since those funds effectively no longer exist

Reply:
Rename to "Burned Total"